src/presentation/pages/Note.vue

```diff
@@ -4,11 +4,17 @@
       :style="{ '--opacity': id && note ? 1 : 0 }"
     >
       <template #left>
-        {{
-          note && 'updatedAt' in note && note.updatedAt
-            ? t('note.lastEdit') + ' ' + getTimeFromNow(note.updatedAt)
-            : t('note.lastEdit') + ' ' + 'a few seconds ago'
-        }}
+        <div>
+          <RouterLink
+            v-for="(parent, index) in displayedParents"
+            :key="index"
+            :to="{ path: `/note/${parent.id ? parent.id : noteId}` }"
+            @click.prevent="handleParentClick($event, parent)"
```

Comment:
probably you don't need to handle click manually since you are using router link